projects/instantbird/0001-Set-Tor-Messenger-preferences.patch

```diff
@@ -1,7 +1,7 @@
 From 5f249d152ab95c0a71ff80cbb4c6b8d6acca2154 Mon Sep 17 00:00:00 2001
 From: Sukhbir Singh <sukhbir@torproject.org>
 Date: Mon, 10 Oct 2016 19:48:41 -0700
-Subject: [PATCH 01/27] Set Tor Messenger preferences
+Subject: [PATCH 01/28] Set Tor Messenger preferences
```

Comment:
Ahh, sorry. Yes, I exported hg to git (already did it since my first build months ago).